InvenTree/InvenTree/urls.py

```diff
@@ -122,6 +125,13 @@
     url(r'^table_filters.js', DynamicJsView.as_view(template_name='js/translated/table_filters.js'), name='table_filters.js'),
 ]
 
+# Integration plugin urls
+interation_urls = []
+if settings.TESTING or InvenTreeSetting.get_setting('ENABLE_PLUGINS_URL'):
```

Comment:
Your call to `InvenTreeSetting.get_setting` will occur when the `settings.py` file is first loaded. 

In the github workflow, the test database is not yet created, and so the table doesn't exist yet! I don't think it is safe to have any functions that access the actual database in the code run by `settings.py`

Reply:
> Calling a non existing setting should return the set default, shouldn't it?

Yes, but it first tries to hit the database (which, if I am reading the issue correctly, isn't ready yet). 

Perhaps a try/except clause around the `get_setting` function?